### PR TITLE
CIV-11543 Submit function for claimant intention

### PR DESCRIPTION
**Pull request by paul-pearson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-11543

### Change description ###

 Add new doc type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ElifDenizEsea — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java [hunk 1]
```diff
@@ -24,6 +25,7 @@ public class TranslatedDocument {
     public DocumentType getCorrespondingDocumentType() {
         switch (documentType) {
             case DEFENDANT_RESPONSE : return DEFENCE_TRANSLATED_DOCUMENT;
+            case CLAIM_INTENTION : return CLAIM_INTENTION_TRANSLATED_DOCUMENT;
```
Comment (manish14836) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java:28]: please check if it should be "CLAIMANT_INTENTION" or "CLAIM_INTENTION" here and all other places

Comment (paul-pearson) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java]: Good spot Manish.  Confirmed with Kiera it should be CLAIMANT_INTENTION.  The ticket was incorrect.

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType;
 
 import static uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType.DEFENCE_TRANSLATED_DOCUMENT;
+import static uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType.CLAIM_INTENTION_TRANSLATED_DOCUMENT;
```
Comment (manish14836) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java:13]: this should be CLAIMANT_INTENTION_TRANSLATED_DOCUMENT

**Review by manish14836 — CHANGES_REQUESTED** [review]

**Review by manish14836 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java (hunk 2 above)
Comment (paul-pearson) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java]: Fixed

**Pull request closed without merging**